python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -52,6 +53,50 @@ def _load_manifest_for_project(
         return json.load(f), cli_output
 
 
+def _select_unique_ids_from_manifest_json(
+    manifest_json: Dict[str, Any], select: str
+) -> AbstractSet[str]:
+    """Method to apply a selection string to an existing manifest.json file."""
+    try:
+        import dbt.graph.cli as graph_cli
+        import dbt.graph.selector as graph_selector
+        from dbt.contracts.graph.manifest import Manifest
+        from networkx import DiGraph
```

Comment:
Is networkx a dagster-dbt dependency? Might people find this error confusing if the problem is that networkx is missing?

Reply:
oh i misread, I'll update the error message